src/thridParty/Tess4J/build.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
* the thirdparty libraries shouldn't be in src folder. I think we can create a folder outside of src e.g.: /thirdparty/Tess4J
* I think we mustn't copy these libraries to the repository. we need to find another way to get it. or we should assume that this thirdparty software is going to be copied by jenkins job